app/routes/users/components/UserConfirmation.js

```diff
@@ -44,9 +44,15 @@ const UserConfirmation = ({
             <div>
               <h2>Du er nå registrert!</h2>
               <h3 style={{ margin: 0 }}>Er du student?</h3>
+              <p className={styles.infoText}>
+                For å kunne melde deg på arrangementer i Abakus, deriblant
```

Comment:
Hmm, that's true. I did think about it after I wrote it but decided it does more good than bad that people validate their profile early. Although it might lead to some confusion :thinking:.

Reply:
While this is not a big deal, some people might get stressed out if they read this and are unable to validate their account for some reason. Especially if they register before the semester starts.

Reply:
Totally agree, I'll remove the part with fadderperioden :)